bigbluebutton-html5/imports/api/users-infos/server/modifiers/clearUserInfoForRequester.js

```diff
@@ -0,0 +1,8 @@
+import UserInfos from '/imports/api/users-infos';
+import Logger from '/imports/startup/server/logger';
+
+export default function clearUsersInfoForRequester(meetingId, requesterUserId) {
+  return UserInfos.remove({ meetingId }, () => {
```

Comment:
Do we need a case where we flush the entire collection? Like on meteor [re]start -- what we do for other collections

Reply:
I don't think we actually use that for other collections.